Assets/Scripts/EnemyController.cs

```diff
@@ -16,130 +18,120 @@ public class EnemyController : MonoBehaviour
     public int maxSpeed;
     public int shieldRegen;
     public float weight;
-    public string movementType;
+    public MovementType movementType;
     public Vector3 movementTarget;
     public List<Transform> path; // hierarchy objects should be ordered
     public int currentPathTarget = 0;
     public float distanceTolerance;
-
+    public float speed = 0.05f;
 
     private Rigidbody2D enemyRigidBody;
+
+    // START Unity magic functions
```

Comment:
Whenever I've used unity, I always get confused as to which functions are the "magic" unity functions like Start() and Update(), so I always make a comment block of START and END markers where in between I put all of (and only) the magic Unity MonoBehavior hook functions. I also as personal preference leave the magic functions as the only ones without a function access modifiers (public, private, internal, etc) so they're also easy to identify.